bluesky_widgets/models/run_engine_client.py

```diff
@@ -50,15 +50,43 @@ class RunEngineClient:
     def __init__(self, worker_address=None):
         self._client = ZMQCommSendThreads(zmq_server_address=worker_address)
 
+        # User name and group are hard coded for now
```

Comment:
Can you expose these as parameters in `__init__` with default values so that when the time comes it's easier for someone to reach in and change them?

Reply:
Yes, I will expose the parameters.

Reply:
I added user name and user group as parameters. This should be sufficient for prototyping.